Content/Items/Magnet/Weapons.ThunderBeads.cs

```diff
@@ -0,0 +1,280 @@
+﻿using log4net.Core;
+using ReLogic.Content;
+using StarlightRiver.Helpers;
+using System;
+using System.Collections.Generic;
+using Terraria.Audio;
+using Terraria.GameContent.Creative;
+using Terraria.ID;
+using Terraria.Graphics.Effects;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement.TaskbarClock;
+using StarlightRiver.Content.Dusts;
+using System.Runtime.InteropServices;
+
+namespace StarlightRiver.Content.Items.Magnet
+{
+	public class ThunderBeads : ModItem
+	{
+		public override string Texture => AssetDirectory.MagnetItem + Name;
+
+		public override void SetStaticDefaults()
+		{
+			DisplayName.SetDefault("Magnet Beads");
+			Tooltip.SetDefault("Whip enemies to stick the beads in them \nRepeatedly click to shock affected enemies");
```

Comment:
We could probably do with some less suggestive wording for this...

Reply:
Fixed